Bottom: 0eb130b4ba359ebb23009327f88447b508a575ab
Top:    c881c2618ad83e3c43a73e6f27af18b7e1124f98
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-19 15:50:12 +0800

Refresh of virtio-net-reset-num_buf-to-1

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index 910d2e7..d9637ec 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -536,9 +536,7 @@ static struct page *xdp_linearize_page(struct receive_queue *rq,
 		put_page(p);
 	}
 
-	/* receive_mergeable() assumes there's at least one buffer, so
-	 * align with it.
-	 */
+	/* receive_mergeable() assumes there's at least one buffer */
 	*num_buf = 1;
 	/* Headroom does not contribute to packet length */
 	*len = page_off - VIRTIO_XDP_HEADROOM;
